index.js

```diff
@@ -88,9 +102,10 @@ datasets.forEach(dataset => {
         activityData: ['userID', 'date', 'flightsOfStairs', 'minutesActive', 'numSteps'],
         hydrationData: ['userID', 'date', 'numOunces'],
         bookings: ['userID', 'date', 'roomNumber'],
-        roomServices: ['userID', 'date', 'food', 'totalCost']
-      };
-
+        roomServices: ['userID', 'date', 'food', 'totalCost'],
+        destinations: ['id', 'destination', 'estimatedLodgingCostPerDay', 'estimatedFlightCostPerPerson', 'image', 'alt'],
```

Comment:
Same thing here. The POSTs are working, but they work if the destination's ID already exists in the dataset. 
The request also still gets added if you add an extra parameter

POST with extra parameter:
<img width="1106" alt="Screen Shot 2020-02-10 at 10 43 52 AM" src="https://user-images.githubusercontent.com/36432040/74175124-66f93b80-4bf2-11ea-9803-197c8b3a8dca.png">

resulting GET request:
<img width="633" alt="Screen Shot 2020-02-10 at 10 43 36 AM" src="https://user-images.githubusercontent.com/36432040/74175175-809a8300-4bf2-11ea-984c-9f3de0ce0b9b.png">